MediaModelBundle/Repository/MediaRepository.php

```diff
@@ -51,7 +51,12 @@ public function findByKeywords($keywords)
     {
         $qb = $this->createQueryBuilder();
 
-        $qb->field('keywords.label')->in(explode(',', $keywords));
+        $keywords = explode(',', $keywords);
+        foreach($keywords as $key => $keyword) {
+            $keywords[$key] = new \MongoId($keyword);
```

Comment:
@itkg-ngilain $keyword is the id and new MongoId($keyword) is just a structure to query the database.